src/preset_cli/cli/superset/sync/dbt/lib.py

```diff
@@ -328,9 +331,18 @@ def filter_models(models: List[ModelSchema], condition: str) -> List[ModelSchema
 
     if condition.startswith("config"):
         filtered_models = []
-        config_key, config_value = re.split(r"[.:]", condition)[1:]
+        config_key, _, config_value = condition.rpartition(":")
+        parts = config_key.split(".")
         for model in models:
-            if model.get("config", {}).get(config_key) == config_value:
+            config = model.get("config", {})
+
+            if len(parts) == 2:  # config.key:value
```

Comment:
What about `config.foo.bar.baz`? We could make this work with any number of parts, something like:

```python
value = model
while parts:
    part = parts.pop(0)
    value = value.get(part, {})

if value == config_value:
    filtered_models.append(model)
```

Reply:
alright, done! 🙌